### PR TITLE
[cm_be_17] Create contract for logout with a placeholder implementation

### PR DESCRIPTION
**Pull request by ajayvk-tw**

### CODE REVIEW

File: src/main/resources/static/api.yaml [hunk 1]
```diff
@@ -1207,6 +1207,33 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+  # Sessions
+  /logout:
+    post:
+      description: |
+        Logout active user by invalidating the access and refresh tokens
+      consumes:
+        - application/x-www-form-urlencoded
```
Comment (ganesanarun) [inline src/main/resources/static/api.yaml:1216]: what makes you think that we should use url-encoded?

Comment (ajayvk-tw) [reply src/main/resources/static/api.yaml:1216]: I don't foresee any new param to be added. Hence found form body to be convenient for parsing without adding a model object.

Comment (ajayvk-tw) [reply src/main/resources/static/api.yaml:1216]: I actually meant `form data` not `form-urlencoded` . Will update if you're okay with it.

Comment (ajayvk-tw) [reply src/main/resources/static/api.yaml]: Updated to json

**Pull request merged**